bonsai/examples/governance/methods/guest/Cargo.toml

```diff
@@ -8,10 +8,11 @@ name = "finalize_votes"
 path = "src/bin/finalize_votes.rs"
 
 [dependencies]
+risc0-zkvm = { version = "0.16", path = "../../../../../risc0/zkvm", default-features = false, features = ["std"] }
```

Comment:
Please sort deps

Reply:
Done in cf1c802c4328f19f3d6d32961dd985c1d6acb463